src/Tests/Tests/Document/Multiple/MultiTermVectors/MultiTermVectorsApiTests.cs

```diff
@@ -123,4 +123,84 @@ private static void AssertTermVectors(TermVector vectors)
 				})
 		};
 	}
+
+	public class MultiTermVectorsIdsApiTests : ApiIntegrationTestBase<ReadOnlyCluster, IMultiTermVectorsResponse, IMultiTermVectorsRequest, MultiTermVectorsDescriptor, MultiTermVectorsRequest>
+	{
+		public MultiTermVectorsIdsApiTests(ReadOnlyCluster cluster, EndpointUsage usage) : base(cluster, usage) { }
+		protected override LazyResponses ClientUsage() => Calls(
+			fluent: (client, f) => client.MultiTermVectors(f),
+			fluentAsync: (client, f) => client.MultiTermVectorsAsync(f),
+			request: (client, r) => client.MultiTermVectors(r),
+			requestAsync: (client, r) => client.MultiTermVectorsAsync(r)
+		);
+
+		protected override bool ExpectIsValid => true;
+		protected override int ExpectStatusCode => 200;
+		protected override HttpMethod HttpMethod => HttpMethod.POST;
+		protected override string UrlPath =>
+			$"/devs/developer/_mtermvectors?field_statistics=true&payloads=true&term_statistics=true&positions=true&offsets=true";
+
+		protected override bool SupportsDeserialization => false;
+
+		protected override object ExpectJson { get; } = new
+		{
+			ids = Developer.Developers.Select(p => (Id)p.Id).Take(2)
```

Comment:
Balmer-object orientation.

Reply:
I thought about providing overloads for `string`, `long`, `Guid`, etc. much like the existing `GetMany<T>(...)`.

Reply:
Could be useful, but easy to add later :)

Reply:
that's what I err'ed on!